app/assets/javascripts/sprangular/services/catalog.coffee

```diff
@@ -27,15 +27,15 @@ Sprangular.service 'Catalog', ($http, $q, _, Status, Env) ->
         return result
 
     taxon: (path) ->
-      $http.get("/api/taxons/#{path}")
+      $http.get("/api/taxons/#{path}", {cache: true})
         .then (response) ->
           response.data
 
     find: (id) ->
-      $http.get("/api/products/#{id}", class: Sprangular.Product)
+      $http.get("/api/products/#{id}", class: Sprangular.Product, {cache: true})
 
     getPaged: (page=1, params={}) ->
-      $http.get("/api/products", ignoreLoadingIndicator: params.ignoreLoadingIndicator, params: {per_page: @pageSize, page: page, "q[name_or_description_cont]": params.search, "q[taxons_permalink_eq]": params.taxon})
+      $http.get("/api/products", ignoreLoadingIndicator: params.ignoreLoadingIndicator, params: {per_page: @pageSize, page: page, "q[name_or_description_cont]": params.search, "q[taxons_permalink_eq]": params.taxon}, {cache: true})
```

Comment:
@simmer noticed that anything with `cache: true` is never cleared. We may
need to add code to wipe cache on bootup.

e.g. `Sprangular.run($cacheFactory) -> $cacheFactory.removeAll()`

On Wed, Mar 18, 2015 at 11:03 AM, Hound <notifications@github.com> wrote:

> In app/assets/javascripts/sprangular/services/catalog.coffee
> <https://github.com/sprangular/sprangular/pull/112#discussion_r26670850>:
>
> >
> >      getPaged: (page=1, params={}) ->
> > -      $http.get("/api/products", ignoreLoadingIndicator: params.ignoreLoadingIndicator, params: {per_page: @pageSize, page: page, "q[name_or_description_cont]": params.search, "q[taxons_permalink_eq]": params.taxon})
> > +      $http.get("/api/products", ignoreLoadingIndicator: params.ignoreLoadingIndicator, params: {per_page: @pageSize, page: page, "q[name_or_description_cont]": params.search, "q[taxons_permalink_eq]": params.taxon}, {cache: true})
>
> Line exceeds maximum allowed length
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sprangular/sprangular/pull/112/files#r26670850>.
>



-- 
Joshua Nussbaum